### PR TITLE
REP-5522 Brought over the updates from a custom filter into the main baseline.

### PR DESCRIPTION
**Pull request by wdschei**

### CODE REVIEW

**Comment by wdschei** [comment]
Retest this please.

**Comment by dmnjohns** [comment]
Retest this please.

Sorry Bill, I killed this one on accident.

**Comment by adrianjgeorge** [comment]
retest this please

**Comment by adrianjgeorge** [comment]
@wdschei this seems super broke

File: repose-aggregator/tests/functional-test-framework/src/main/groovy/org/openrepose/framework/test/mocks/MockIdentityV2Service.groovy [hunk 1]
```diff
@@ -758,7 +761,7 @@ class MockIdentityV2Service {
         def tenantId = values.tenantId ?: client_tenantid
         def userId = values.userId ?: client_userid
         def username = values.username ?: client_username
-        def roleNames = values.roles ?: ["identity:admin"]
+        def roles = (values.roles ?: [["identity:admin"]])
```
Comment (Mario-Lopez) [inline repose-aggregator/tests/functional-test-framework/src/main/groovy/org/openrepose/framework/test/mocks/MockIdentityV2Service.groovy]: Should be:
```
def roles = (values.roles ?: [[name: "identity:admin"]])
```

Comment (wdschei) [inline repose-aggregator/tests/functional-test-framework/src/main/groovy/org/openrepose/framework/test/mocks/MockIdentityV2Service.groovy]: Updated.

**Comment by wdschei** [comment]
@adrianjgeorge The fix that @Mario-Lopez posted was the issue. This is ready for review again.

**Pull request merged**